plugins/azure/keyvaults/cmkCreationForAppTier.js

```diff
@@ -0,0 +1,86 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+var _ = require('underscore');
+
+module.exports = {
+    title: 'CMK Creation for App Tier Enabled',
+    category: 'Key Vaults',
+    domain: 'Identity and Access Management',
```

Comment:
Can you make this 'Application Integration' for this and other key vault plugins in same PR